website/docs/guides/sl-cookbook-recipes.md

```diff
@@ -0,0 +1,418 @@
+
+<File name="models/semantic_models/sem_user_activity_daily.yml">
+
+```yaml
+semantic_models:
+  - name: sem_user_activity_daily
+    description: >
+      This model reports on daily user activity. Activity is broken down into
+      pageview + time-on-site metrics for each use-case that is addressed in
+      dbt Cloud (ie. Deployment and Development)
+    model: ref('fct_user_activity_daily')
+    defaults:
+      agg_time_dimension: date_day
+      
+# Newly added blocks below 
+
+    entities:
+    #--- user entities
+      - name: user_daily
+        type: primary
+        expr: user_activity_daily_id
+      - name: user
+        type: foreign
+        expr: user_id
+      #--- util entities
+      - name: date
+        type: foreign
+        expr: date_day
+
+    dimensions:
+      - name: date_day
+        type: time
+        type_params:
+          time_granularity: day
+     - name: t7d_usage
+        type: categorical
+
+    measures:
+      - name: count_activity_users
+        description: >
+          Distinct count of users via `fct_user_activity_daily`.
+          To be used in metrics related to User-level activities
+          (e.g. Active Users)
+        agg: count_distinct
+        expr: user_id
+        agg_time_dimension: date_day
+```
+</File>
+
+3. Create another semantic model YAML file (`models/semantic_models/sem_all_days.yml`). This semantic model captures company-specific fiscal periods and date flags (like `fiscal_quarter`). 
+  
+  It helps in managing fiscal dates and other time-related dimensions.
+
+<File name="models/semantic_models/sem_all_days.yml">
+
+```yaml
+- name: sem_all_days
+    description: >
+      Utility semantic model to bring in company-specific fiscal quarter and fiscal year, or date flags such as `is_end_of_week` or `is_end_of_fiscal_quarter`.
+    model: ref('all_days')
+    defaults:
+      agg_time_dimension: date_day
+
+    entities:
+      - name: date
+        type: primary
+        expr: date_day
+      - name: date_month
+        type: foreign
+        expr: month_start_date
+
+    dimensions:
+      - name: date_day
+        type: time
+        type_params:
+          time_granularity: day
+      - name: date_month
+        type: time
+        type_params:
+          time_granularity: month
+      - name: fiscal_quarter
+        type: categorical
+      - name: fiscal_year
+        type: categorical
+      - name: is_end_of_week
+        type: categorical
+      - name: is_yesterday
+        type: categorical
+```
+</File>
+
+### Step 2: Create metrics
+
+1. Create a YAML file under the `metrics` folder: (`models/metrics/sem_user_activity_daily.yml`) to create a metric that tracks active users. 
+   
+  Note that if you’re using the one-YAML-file-per-mart, then save this as `active_users.yml` under the models folder.
+
+<File name="models/metrics/sem_user_activity_daily.yml">
+
+```yaml
+metrics:
+  - name: users_active
+    label: 'Users: active'
+    description: >
```

Comment:
I've been encouraging the Data team to use `|` over `>` as the `|` takes newlines into consideration within the markdown snippet.
```suggestion
    description: |
```